de-lib/src/main/java/org/iplantc/de/client/models/UserSettings.java

```diff
@@ -272,12 +274,14 @@ public Splittable asSplittable() {
         return ret;
     }
 
-    public void useDefaultValues(UserInfo instance) {
+    public void useDefaultValues(UserInfo userInfo) {
         JSONObject defaults = new JSONObject();
```

Comment:
I noticed that too, and that `Splittable` is already used a lot in `UserSettings`, but why not use an `AutoBean` instead?

This seems like something for another PR.